packages/scaffold-html/src/views/w3m-what-is-a-network-view/index.ts

```diff
@@ -0,0 +1,73 @@
+import { LitElement, html } from 'lit'
+import { customElement } from 'lit/decorators.js'
+import styles from './styles'
+import { VisualType } from '@web3modal/ui/src/utils/TypesUtil'
+import { RouterController } from '@web3modal/core'
+
+type Data = {
+  images: VisualType[]
+  title: string
+  text: string
+}
+
+const data: Data[] = [
```

Comment:
`Data` type is probably not necessary here. If it needs to be here to satisfy something further down below, you can add `as const` at the end of the data array.